src/backend/TrafficCourtsApi/citizen-api-test/Features/Lookups/LookupControllerTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading;
+using AutoFixture.Xunit2;
+using Gov.CitizenApi.Features.Lookups;
+using Gov.CitizenApi.Features;
+using Gov.CitizenApi.Features.Disputes;
+using Gov.CitizenApi.Features.Disputes.Commands;
+using Gov.CitizenApi.Features.Disputes.Queries;
+using Gov.CitizenApi.Test.Utils;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+using System.Threading.Tasks;
+
+namespace Gov.CitizenApi.Test.Features.Lookups
+{
+    [ExcludeFromCodeCoverage]
+    public class LookupControllerTest
+    {
+        private readonly Mock<ILogger<LookupController>> _loggerMock;
+        private readonly Mock<ILookupsService> _serviceMock;
+
+        public LookupControllerTest()
+        {
+            _loggerMock = LoggerServiceMock.LoggerMock<LookupController>();
+            _serviceMock = new Mock<ILookupsService>();
+        }
+
+    [Fact]
+#pragma warning disable IDE1006 // Naming Styles
+        public void throw_ArgumentNullException_if_passed_null()
+    #pragma warning restore IDE1006 // Naming Styles
+        {
+            Assert.Throws<ArgumentNullException>(() => new LookupController(null, _serviceMock.Object));
+            Assert.Throws<ArgumentNullException>(() => new LookupController(_loggerMock.Object, null));
+        }
+
+        [Fact]
+#pragma warning disable IDE1006 // Naming Styles
+        public async Task get_lookups()
```

Comment:
name the test something meaningful. When reviewing a failed test, what would I think when I see get_lookups failed?